src/Reflection/Term.agda

```diff
@@ -49,6 +54,66 @@ pattern vΠ[_∶_]_ s a ty    = Π[ s ∶ (vArg a) ] ty
 pattern hΠ[_∶_]_ s a ty    = Π[ s ∶ (hArg a) ] ty
 pattern iΠ[_∶_]_ s a ty    = Π[ s ∶ (iArg a) ] ty
 
+------------------------------------------------------------------------
+-- Showing
+
+-- Note that Reflection.termErr can also be used directly in tactic error
+-- messages.
+
+private
+  open ListMembership CP._≟_
+
+  -- enclose string with parens if it contains a space character
+  parensIfSpace : String -> String
+  parensIfSpace s with ' ' ∈? String.toList s
+  ...             | yes _ = parens s
+  ...             | no _  = s
```

Comment:
Can we get rid of the spaces between the `...` and the `|`. Sorry I know it seems minor. Really looking forward to the day when someone keen comes up with a linter for Agda!

Reply:
Of course, thanks for flagging it up!